mantidproject/mantid
# Load new D2B NeXus files

The NeXus files for D2B have been updated* to provide calibrate data. This is stored in `data_scan/detector_data/data`, with the presence of the entry `data_scan/detector_data/raw_data` telling you the other data block is calibrated data.

Some scanned variables were also added with the same name, so the name when loaded into Mantid is now `name.property`.

*The NeXus files are not being used yet so they are still changed sometimes.

**To test:**

Check code and unit test logic. Try out the attached D2B and D20 files, check both load with no `DataType` given, the D2B loads for `Calibrated` or `Raw` and the D20 file loads for `Calibrated` and throws for `Raw`.

[example_data.zip](https://github.com/mantidproject/mantid/files/1440578/example_data.zip)

Fixes #20881.

**Release Notes** 

Updated.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
